### ISSUE TITLE
Questions about: Transmutations of metadata

### ISSUE THREAD

**Issue by augustfly** [augustfly/DOI4Archives#8]
@gilleslandais originally wrote by email: 

> (8) Transmutations of metadata?
>   there is also the "supplying metadata in multiple formats" - do you mean the "format" Datacite term? or content-type available in landing page ?

**Comment by augustfly** [comment]
https://github.com/augustfly/DOI4Archives/blob/ad3864ebd4049201848d3035c68c33857ea8ab40/DOI4Archives.tex#L188-L191

I do not mean the Datacite term, "format" which refers to the "Technical format of the resources". I interpret [DataCite Schema 4.4](https://doi.org/10.14454/3w3z-sa82) to mean that this field gives a hint as to the mime-type of the resource provided. 

I mean that a dataset provider may wish to provide the same metadata in multiple formats: submitted to DataCite [1]; displayed as (visible) HTML on the landing page; embedded as schema.org tags in the landing page. Data providers **must** ensure that all formats are consistent. This is one of the worst pathologies out there today.

[1] Note that DataCite does require a specific "format" of metadata submission. A data provider could submit metadata to DataCite in Schema.org and DataCite would transmutate (via their crosswalk) to load the DataCite record. This means that a data provider does not need to subscribe purely to DataCite XML. Just that their adoption of Schema.org produces consistently transmutated (crosswalked) datacite metadata.

**Comment by gilleslandais** [comment]
the list of formats available for creation evolved - I ignored that schmea.org was now available-

Good idea to highlight crosswalk and  provide consistent metadata!

**Comment by BaptisteCecconi** [comment]
Beware that the various metadata schema may not have the same scope, to the mappings and crosswalks can't be complete. It should be clear for us between DOI metadata and VOResource metadata, but the same applies to schema.org or DCAT.

**Comment by augustfly** [comment]
Yes, but there is a DataCite to schema.org crosswalk, no? We should emphasize when crosswalks exist so that repositories can make informed choices about what formats they should use to express metadata. I feel this was part of the problem with the initial ESA DOI effort, where creating different metadata expressions and not using a crosswalk leads to different metadata discovery outcomes.

**Comment by BaptisteCecconi** [comment]
yes I agree with this. if a crosswalk exist and is already adopted, then we should advertise and use it so that interoperability is enabled.
